plugins/org.jkiss.dbeaver.model.ai/src/org/jkiss/dbeaver/model/ai/gpt3/GPTCompletionEngine.java

```diff
@@ -174,21 +178,26 @@ private String requestCompletion(
         if (monitor.isCanceled()) {
             return "";
         }
-        CompletionRequest completionRequest = createCompletionRequest(modifiedRequest);
+
+        Object completionRequest = createCompletionRequest(modifiedRequest);
```

Comment:
Do we really need to support legacy API?

Reply:
Yes, because it is used in other models, except turbo.